php/php-k8s-v7/etc/nginx/nginx.conf

```diff
@@ -119,11 +119,17 @@ http {
       default   $upstream_http_x_xss_protection;
       ""        "1; mode=block";
    }
-   add_header X-XSS-Protection $xssprotection;
+  add_header X-XSS-Protection $xssprotection;
 
   ## Block MIME type sniffing on IE.
   add_header X-Content-Options nosniff;
 
+  ## Enable HSTS.
+  add_header Strict-tRansport-Security "max-age=31536000; includeSubDomains; preload; always;";
```

Comment:
"Transport" in this and the other versions.

Reply:
🤦‍♂️